cmd/server-main.go

```diff
@@ -272,36 +272,9 @@ func initAllSubsystems(newObject ObjectLayer) (err error) {
 	// are modifying this code that you do so, if and when
 	// you want to add extra context to your error. This
 	// ensures top level retry works accordingly.
-	var buckets []BucketInfo
```

Comment:
The reason it exists here because we rely on buckets to be present to load the related bucket configs if they are not present then the configs are not updated correctly which means that we cannot reload them until next restart.